contracts/script/deploy/verifiable/doc/DEPLOYMENT_PROCEDURE.md

```diff
@@ -0,0 +1,30 @@
+# Deployment Procedure
+
+This document details the deployment procedure using the `DeployVerifiable.sol` script. Verifying the correctness of the contracts is out of the scope of this document. The process of deploying EigenDA contracts with this script is divided into the following distinct phases:
+
+## Deployment
+
+* Use placeholder.config.json as a template to create a configuration file with all parameters that the contracts need to be initialized with. 
+* Any deployer can run the deployment script with `forge script DeployVerifiable` with the DEPLOY_CONFIG_PATH env variable pointing to the config file. Make sure to use the `--verify` flag to verify the contracts on etherscan.
```

Comment:
might be easier to provide an example command that someone can copy into their CLI vs stitching it together from this description